### ISSUE TITLE
EmojiOne Area pulls libraries from cdnjs.cloudflare.com

### ISSUE THREAD

**Issue by clarkwinkelmann** [clarkwinkelmann/flarum-ext-emojionearea#1]
This is probably due to the default configuration of the library.

It fetches images at `cdnjs.cloudflare.com` whereas the base emoji plugin fetches them at `cdn.jsdelivr.net`.

It also load additional `css` and `js` files from `cdnjs.cloudflare.com`. These files are probably already included in the core and should not be fetched again.

**Comment by clarkwinkelmann** [comment]
`emojione.min.js` is still fetched from cdnjs. It seems this file is not loaded by Flarum core or emoji picker so that's the reason it is loaded.

Emojione Area does not seem to have a way to change the CDN for that action :angry: 

https://github.com/mervick/emojionearea/blob/master/src/function/loadEmojione.js#L45

**Comment by plista-mp** [comment]
yes, would be great to deactivate that part if the option "useInternalCDN" is deactivated
